crypto/rand/rand_unix.c

```diff
@@ -128,7 +128,7 @@ int RAND_poll(void)
 #   error "librandom not (yet) supported"
 #  endif
 
-int RAND_poll(void)
+int RAND_poll_ex(RAND_poll_cb cb, void *arg)
```

Comment:
I'm about to push a commit that re-orders and returns after the first success, thanks!